projects/packages/jetpack-mu-wpcom/src/features/launch-button/index.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Adds a "launch site" button to the admin bar.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+use Automattic\Jetpack\Status;
+
+/**
+ * Adds a "launch site" button to the admin bar.
+ *
+ * @param WP_Admin_Bar $admin_bar The WordPress admin bar.
+ */
+function wpcom_add_launch_button_to_admin_bar( WP_Admin_Bar $admin_bar ) {
```

Comment:
We need to check whether the user is the admin of the site. Currently, the launch button shows up for everyone, even logged-out users, which is unexpected.

Reply:
This is how we can reproduce the issue:

1. Create a new unlaunched SIMPLE site.
2. Patch this PR to the sandbox.
3. Add a new user as Editor/Author.
4. Log in to WordPress.com as that user (e.g. in another browser window).
5. Open the site frontend as that user.
6. You will see the admin bar with the launch button, which shouldn't be there. (Clicking the button will not launch the site)

> Isn't the admin bar only shown for logged-in users.

Yeah, sorry, it won't be shown logged-out users. (Except if you're seeing an ATOMIC site frontend while PROXIED, which is a weird case and can be ignored). But the above case still applies -- we need to check for the current user's permission.

Reply:
So, I was actually unable to reproduce this issue. The Editor/Author can't see the frontend of the website and only see a "coming soon" page for me, so there's no admin bar. 

That said, I pushed a code to add a check. Some small defensive coding is not that bad.